packages/core/lib/debug/interpreter.js

```diff
@@ -761,6 +765,21 @@ class DebugInterpreter {
     }
     return tempPrintouts;
   }
+
+  // parse the numbers of lines options -<num>|+<num> from user args
+  parsePrintoutLines(userArgs, currentLines) {
+    for (const argument of userArgs) {
+      if (argument.length < 2) continue;
```

Comment:
It's a little unclear what this line is doing.  I mean, I think I understand -- it's there so that a bare `+` or `-` isn't interpreted as `+0` or `-0` -- but I only picked up on that because I've been dealing a lot with details of number parsing lately.  I think you should add a comment here to clarify that; other readers won't get that.  Or, if I've misunderstood what it's doing, then it *definitely* needs a comment! :P

Reply:
I will ad a comment here for clarity, thx.

Reply:
Thank you!